common/openapi.yml

```diff
@@ -502,6 +502,24 @@ components:
       type: string
       format: string
 
+    PipelineInputName:
+      description: |
+        The name of the input field.
+      type: string
+      format: string
+
+    PipelineInputType:
+        description: |
+            The type of the input field.
+        type: string
+        format: string
+
+    PipelineInputIsRequired:
```

Comment:
i think this one should go first alphabetically (i kinda hate myself for pointing this out)

Reply:
ha! you're totally right - thank you!